_generated/AzureRmWebAppDeploymentV4.versionmap.txt

```diff
@@ -1,2 +1,2 @@
 Default|4.232.2
-Node16-225|4.232.3
+Node20_229_2|4.232.3
```

Comment:
looks like there is a loophole in the tooling which is allowing a version to get re-used (4.232.3).  To be safe, please bump the base task.  (same with IISWebAppDeploy)

Reply:
Updated, versions bumped up.